CONTRIBUTING.md

```diff
@@ -1,12 +1,15 @@
 # Contributing guidelines
 
+This document has the guidelines and the required information for who would like to contribute to this project. Currently, Kubebuilder officially supports OSX and Linux platforms, Windows is still not fully supported so you may find issues to execute some of the following commands under this OS.
```

Comment:
I don't know that the first sentence adds anything that the reader doesn't already know (ie. I'm looking at the contributing guidelines because I want information about contributing to the project). The second sentence I would expect to find in the main project README, not here.

Reply:
A small descrition as the first sentence is ok from my point of view. About the second sentence, I definetely agree that it shoul me in the README, but it could also be here.

Reply:
The first one clarifies the purpose of the doc and for who it is important. 
Moving the second one to the README.

Reply:
I chose my words poorly. The first sentence, to me at least, is redundant to what the document _is_.

By this I mean a "Contributing guidelines" document is exactly that, a document with guidelines and other useful information for those who want to contribute to the project. From my perspective it would be like starting the `README` with a sentence saying "This is a document about xyz project".

It doesn't hurt to have it so consider it a nit. I was just making the point that it was not a substantive change.

Reply:
Done.